.changeset/hot-apples-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': major
+---
+
+Removed experimental suffix from Avatar size values
```

Comment:
Can we also add the size name changes? I think those are more important to document than experimental changes